datalad/crawler/nodes/annex.py

```diff
@@ -350,7 +350,7 @@ def __init__(self, path=None,
         self._statusdb = None  # actual DB to be instantiated later
 
         if ultimatedb is None:
-            ultimatedb = cfg.getboolean('crawl', 'annex.ultimatedb', default=False)
+            ultimatedb = cfg.getbool('crawl', 'annex.ultimatedb', default=False)
```

Comment:
forgot to pull ;)
in general -- shouldn't we use `datalad.` prefix in all `cfg.getbool` etc uses for internal config variables?  sample grep showed that it seems to be the case -- ie for internal - `datalad.` prefix. correct @mih?
```
$> git grep 'cfg\.get\S*(' | grep -v -e oldconfig -e test_
datalad/cmd.py:                    cfg.getbool('datalad.log', 'outputs', default=False)
datalad/distribution/create_sibling_github.py:                github_user = cfg.get('hub.oauthtoken', None)
datalad/log.py:        show_timestamps = anything2bool(cfg.get('datalad.log.timestamp', False))
datalad/support/annexrepo.py:                     cfg.getbool("datalad", "repo.direct", default=False)
datalad/support/annexrepo.py:            self._uuid = repocfg.get_value('annex', 'uuid', default='')
```